src/common/dependencies.ts

```diff
@@ -125,6 +125,34 @@ export const getOptionalDependencies = (isNvidiaAvailable: boolean): Dependency[
             description:
                 'ONNX uses .onnx models to upscale images. It also helps to convert between PyTorch and NCNN. It is fastest when CUDA is supported. If TensorRT is installed on the system, it can also be configured to use that.',
         },
+        {
+            name: 'Stable Diffusion',
+            packages: [
+                {
+                    packageName: 'comfylib',
+                    version: '0.0.5',
+                    sizeEstimate: 3 * GB,
```

Comment:
What exactly makes this 3GB? We aren't downloading torch again through this are we?

Reply:
There is a bunch of other junk that's pulled in, directly or indirectly.  I should do a pass on the comfy code and figure out if there's any dependencies  we can drop.  I believe it should not re-download pytorch, if it's already installed.

With a fresh venv, after installing comfylib the `lib` directory is 4.6GB, and contains these libraries:  https://gist.github.com/adodge/b6722ab4d2229562ef5e2663a85e1daf

Reply:
What's that `nvidia` package?

Reply:
hrm.  I'm not sure.  That doesn't seem to be installed anymore.  it's possible I didn't do that experiment correctly.

Here's the list of dependencies for comfylib, and all their dependencies, courtesy of `pipdeptree`:  https://gist.github.com/adodge/de2e1686f0ec19727c44bd905b3c119e

When I get a chance, I'll go through and do a dropout test to see which of these are actually being used by the code integrated in chainner.

Reply:
When I"m done, I'll add a specific version of all the packages it installs to `dependencies.ts`.